open_schools_platform/organization_management/circles/services.py

```diff
@@ -1,8 +1,7 @@
-from typing import Dict, Callable
+from typing import Dict, Callable, Tuple, Type
```

Comment:
you make a revolution!)
we should implement this logic in BaseQueryHandler, it would be greateful!
but i think we will do it in other branch